dockerfiles/cli/scripts/openshift/deploy_che.sh

```diff
@@ -76,9 +76,19 @@ done
 # Set configuration common to both minishift and openshift
```

Comment:
on a global remark, this script should not be in cli/scripts but probably in base/scripts and fit the existing model on how to define commands

Reply:
I moved it, as well as the other scripts and openshift descriptors to `che-init`

Reply:
@benoitf @davidfestal I would prefer if we you keep this script where it is. This PR is not about fixing the location of `deploy_che.sh`. Fixing that will be handled in another PR on master branch. Moving `deploy_che.sh` to another folder in this PR will make merging `multi-user` branch to `master` more complicated.

Reply:
The PR to move the `deploy_che.sh` script is under review on master  (#6390). As soon as it is merged, and `master` is merged into the `che-multiuser` branch, I'll rebase this PR on `che-multiuser` and things will be clean.

Is it OK for you @l0rd ?